pkg/cli/debug.go

```diff
@@ -793,13 +793,13 @@ func runDebugGCCmd(cmd *cobra.Command, args []string) error {
 
 	var rangeID roachpb.RangeID
 	gcTTL := 24 * time.Hour
-	intentAgeThreshold := gc.IntentAgeThreshold.Default()
-	intentBatchSize := gc.MaxIntentsPerCleanupBatch.Default()
+	lockAgeThreshold := gc.LockAgeThreshold.Default()
+	lockBatchSize := gc.MaxLocksPerCleanupBatch.Default()
 	txnCleanupThreshold := gc.TxnCleanupThreshold.Default()
 
 	if len(args) > 3 {
 		var err error
-		if intentAgeThreshold, err = parsePositiveDuration(args[3]); err != nil {
+		if lockAgeThreshold, err = parsePositiveDuration(args[3]); err != nil {
 			return errors.Wrapf(err, "unable to parse %v as intent age threshold", args[3])
```

Comment:
`"unable to parse %v as lock age threshold"`

Reply:
Done.